course/models.py

```diff
@@ -0,0 +1,32 @@
+from django.db import models
+# import user
+
+# All the comments are to be changed after the PR merge of User and Teacher_course models
+
+
+class ReviewManager(models.Manager):
```

Comment:
What is the advantage of implementing the model manager and not using the methods in the `Review` class?

Reply:
The two functions I wrote contain the filter function and the query result can contain multiple instances
The query is not about a specific instance because of that it is better to use the manager.